include/nbl/builtin/hlsl/cmath.hlsl

```diff
@@ -125,12 +142,21 @@ T hypot(T x, T y)
 template<class T>
 T copysign(T x, T sign_)
 {
-    return sign_ < T(0) ? -x : x;
+    using uint_type = uint32_t; //typename nbl::hlsl::numeric_limits<T>::uint_type;
+    constexpr uint_type m = (1u << 31);
```

Comment:
I want all this to be tested and compile as both C++ and HLSL, you obviously haven't tested

HLSL doesn't have constexpr